### PR TITLE
#174797509 Dotenv setup

### PR DESCRIPTION
**Pull request by elvisiraguha**
#### What does this PR do?
setup dotenv to manage environment variables
#### Description of Task to be completed?
- install dotenv
- setup dotenv  and add .env.example file
#### How should this be manually tested?
- Clone the repo `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend.git`
- Switch your working directory `cd script-heroes-bn-backend`
- Checkout the branch `git checkout ch-setup-dotenv-#174797509`
- Create a .env file in the project root directory
- Copy keys in .env.example file, which is in the project root directory and assign values to those keys.
- To use declared variables, require dotenv at the top of your file `require('dotenv').config()`
- Access environment variable value by using `process.env.KEY_NAME` where KEY_NAME is the variable name.
#### What are the relevant pivotal tracker stories?
[#174797509](https://www.pivotaltracker.com/story/show/174797509)

### CODE REVIEW

**Review by Habinezajanvier — COMMENTED** [review]
Hey @elvisiraguha, we having duplication of file in your branch, **swagger.json**

**Review by jpirumvaa — APPROVED** [review]
Well done

**Comment by elvisiraguha** [comment]
> Hey @elvisiraguha, we having duplication of file in your branch, **swagger.json**

Thank you @Habinezajanvier I got it fixed.

**Review by Habinezajanvier — APPROVED** [review]
Great

**Review by dushimeemma — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,13 +12,22 @@ This is the swagger API [documentation](https://script-heroes-bn-backend-stgng.h
...
 This the heroku staging Api https://script-heroes-bn-backend-stgng.herokuapp.com/
 
-This the production Api   https://script-heroes-bn-backend.herokuapp.com/
----
+## This the production Api https://script-heroes-bn-backend.herokuapp.com/
 
+## Setup Dotenv
+
+- Create .env file in project root directory
+- Copy keys in .env.example file, which is in the project root directorym and assign values to those keys.
```
Comment (kipropbrian) [inline README.md]: Spelling issue here, `directorym `

File: README.md [hunk 2]
```diff
@@ -12,13 +12,22 @@ This is the swagger API [documentation](https://script-heroes-bn-backend-stgng.h
 
 This the heroku staging Api https://script-heroes-bn-backend-stgng.herokuapp.com/
 
-This the production Api   https://script-heroes-bn-backend.herokuapp.com/
----
+## This the production Api https://script-heroes-bn-backend.herokuapp.com/
```
Comment (kipropbrian) [inline README.md]: Remove the ## from the beginning.

File: README.md [hunk 3]
```diff
@@ -12,13 +12,22 @@ This is the swagger API [documentation](https://script-heroes-bn-backend-stgng.h
...
+- You can add more environmenta variables to the .env file
+- To use declared variables, require dotenv at the top of your file `require('dotenv').config()`
+- Access environment variable value by using `process.env.KEY_NAME` where KEY_NAME is the variable name.
+
+Note: If you make changes that uses environmental variables make sure to add those variables with example values in the .env.example file.
 
 ## Sequelize and Sequelize-cli
 
 # 1 Installing cli
+
```
Comment (kipropbrian) [inline README.md:30]: If you view this file [here](https://github.com/atlp-rwanda/script-heroes-bn-backend/blob/669e4649fd6bdff089931f9db52b92bb18db7369/README.md) , the content under the sequelize heading look a bit too large. Could you fix that.

Comment (elvisiraguha) [reply README.md:30]: Yeah, I got it fixed.

**Review by kipropbrian — APPROVED** [review]

**Pull request merged**